client/app/telecom/telephony/alias/configuration/changeType/telecom-telephony-alias-configuration-changeType.html

```diff
@@ -28,13 +30,12 @@ <h2 data-translate="loading"></h2>
                     <h2 class="widget-presentation-title"
                         data-translate="telephony_alias_config_change_type_title"></h2>
                 </header>
-                <form class="ovh-form-flat"
-                      name="changeTypeForm"
+                <form name="changeTypeForm"
                       data-ng-submit="AliasConfigurationChangeTypeCtrl.changeType();"
                       data-ng-if="!AliasConfigurationChangeTypeCtrl.loading.changing">
                     <div class="form-group mt-3 clearfix">
```

Comment:
By removing `flat-select` directive we can get ridd off `mt-3 clearfix`.

Reply:
Woops, missed that